docs/designers-developers/developers/tutorials/create-block/readme.md

```diff
@@ -20,13 +20,18 @@ You now need to activate the plugin from inside wp-admin plugins page.
 
 After activated, go to the block editor and use the inserter to search and add your new block.
 
+## Prerequisities
```

Comment:
We also would need to update line 5 in this document as it's no longer 100% correct:

> The tutorial includes setting up your development environment, tools, and getting comfortable with the new development model. If you are already comfortable, try the quick start below, otherwise step through whatever part of the tutorial you need.

It might be better to move `Prerequisities` before `Quick Start` as well.

Reply:
Moved it above 👍